deploy/operator.yaml

```diff
@@ -47,3 +45,25 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+          volumeMounts:
+          - mountPath: /var/run/secrets/openshift/serviceaccount
+            name: oidc-token
+          - mountPath: /var/lib/cosi
+            name: socket
+        - name: objectstorage-provisioner-sidecar
+          image: COSI_SIDECAR_IMAGE
+          args:
+          - "--v=5"
+          resources:
+            limits:
+              cpu: "100m"
+              memory: "512Mi"
+          imagePullPolicy: Always
```

Comment:
Hi @romayalon,
I know it was already merged - small question:
Why do we change the default behavior (`IfNotPresent`)?
We have the option to run `noobaa install` with flag `--operator-image`- Would it still try to pull the image?

Reply:
it's a different container, it's not changing the NooBaa operator's container policy